easybuild/easyconfigs/p/Python/Python-3.11.5-custom-ctypes.patch

```diff
@@ -0,0 +1,150 @@
+Override ctypes behavior to enable support for EB provided libraries. 
```

Comment:
Maybe the description can be improved:
- pick up on `$LIBRARY_PATH` rather than `$LD_LIBRARY_PATH`
- also consider `foo` as full name of library in `_findLib_gcc`, not just `libfoo`
- ignore CUDA stub libraries (based on their path matching `cuda.*stubs`)